src/inc/regdisp.h

```diff
@@ -302,10 +302,10 @@ inline void SyncRegDisplayToCurrentContext(REGDISPLAY* pRD)
 {
     LIMITED_METHOD_CONTRACT;
 
-#if defined(_WIN64)
+#if defined(_TARGET_64BIT_)
     pRD->SP         = (INT_PTR)GetSP(pRD->pCurrentContext);
     pRD->ControlPC  = INT_PTR(GetIP(pRD->pCurrentContext));
-#elif defined(_TARGET_ARM_) // _WIN64
+#elif defined(_TARGET_ARM_) // _TARGET_64BIT_
```

Comment:
Nit: Could you please delete the comments on `#elif`s here instead? They are not useful - our typical style is not use them for this pattern.